ts/components/services/OtherServices.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { NativeScrollEvent, NativeSyntheticEvent } from "react-native";
+import ServiceList from "./ServiceList";
+
+type AnimatedProps = {
+  animated?: {
+    onScroll: (_: NativeSyntheticEvent<NativeScrollEvent>) => void;
+    scrollEventThrottle?: number;
+  };
+  paddingForAnimation: boolean;
+  AnimatedCTAStyle?: any;
+};
+
+type Props = AnimatedProps;
+
+/**
+ * A component to render a list of other services.
+ */
+class OtherServices extends React.PureComponent<Props> {
```

Comment:
```suggestion
class ServicesOther extends React.PureComponent<Props> {
```

The other component is called ServicesLocal.

Reply:
@fpersico done in commit 17dc7ae